.swiftlint.yml

```diff
@@ -0,0 +1,12 @@
+
+disabled_rules:
+  - line_length
+  - type_name
+  - valid_docs
+  - empty_count
+  - force_unwrapping
+  - function_body_length
+  - variable_name
```

Comment:
It'd be awesome if Quick contributors could send pull requests to un-disable each of these rules. One pull request per rule would be great. :)